### PR TITLE
Pyqgis updates

### PR DESCRIPTION
**Pull request by ghtmtt**
Update the **Load Layer** chapter with syntax and classes of Q3. No big changes, just some corrections in some methods.

### CODE REVIEW

File: source/docs/pyqgis_developer_cookbook/loadlayer.rst [hunk 1]
```diff
@@ -53,35 +53,39 @@ providers:
...
+      uri = "/path/to/dxffile/file.dxf|layername=entities|geometrytype=Point"
+      vlayer = QgsVectorLayer(uri, "layer_name_you_like", "ogr")
 
 
 .. index::
   pair: Loading; PostGIS layers
 
 * PostGIS database --- data source is a string with all information needed to
-  create a connection to PostgreSQL database. :class:`QgsDataSourceURI` class
+  create a connection to PostgreSQL database. ``QgsDataSourceUri`` class
```
Comment (DelazJ) [inline source/docs/pyqgis_developer_cookbook/loadlayer.rst]: actually, I think you should update the class list in conf.py file (see https://github.com/qgis/QGIS-Documentation/blob/master/source/conf.py#L1330 - I don't know what it's designed for; @rduivenvoorde might know better)

File: source/docs/pyqgis_developer_cookbook/loadlayer.rst [hunk 2]
```diff
@@ -235,40 +239,40 @@ Alternatively you can load a raster layer from WMS server. However currently
...
-  QgsMapLayerRegistry.instance().removeMapLayer(layer_id)
+    QgsProject.instance().removeMapLayer(layer_id)
 
 For a list of loaded layers and layer ids, use:
 
-::
+.. code-block:: python
 
-  QgsMapLayerRegistry.instance().mapLayers()
+    QgsMapLayQgsProjecterRegistry.instance().mapLayers()
```
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/pyqgis_developer_cookbook/loadlayer.rst]: ahahahah damn copy/paste. Glad you are reviewing so good!

**Comment by ghtmtt** [comment]
@DelazJ I rebased the branch with @SrNetoChan work of the image preview (and forced the push).

File: source/docs/pyqgis_developer_cookbook/loadlayer.rst [hunk 3]
```diff
@@ -20,11 +20,11 @@ Vector Layers
 To load a vector layer, specify layer's data source identifier, name for the
 layer and provider's name:
 
-::
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/pyqgis_developer_cookbook/loadlayer.rst]: mmm ok.. I prefer to make it explicit where possible. Moreover, where can I find the default code-block for the doc? I did not find it in `conf.py`

[the remark this answers is not in the document]
Comment (DelazJ) [reply source/docs/pyqgis_developer_cookbook/loadlayer.rst]: I unsuccessfully looked for it in the conf file yesterday when reviewing this PR and ended up sending the link to sphinx doc's --> if we do not have defined one, then it's the default of the Sphinx that applies.

**Comment by DelazJ** [comment]
LGTM

File: source/docs/pyqgis_developer_cookbook/loadlayer.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (ghtmtt) [reply source/docs/pyqgis_developer_cookbook/loadlayer.rst]: Good. If it is ok, also for other PR I'll explicit use the `.. code-block::` directive: IMHO easier to understand for doc writers and hopefully more clean (I have python code highlighted as python also in my text editor).

**Pull request merged**